spec.bs

```diff
@@ -296,21 +309,47 @@ To <dfn>look up the key commitments</dfn> for a given [=/origin=] |issuer|, run
 
 Note: |pstVersion| is a string identifier representing different cryptographic versions of tokens that can be used with this API. User agents should only select keys for versions they support, ordered by which versions they prefer based on performance and any user defined preferences.
 
+To <dfn>look up the latest keys</dfn> for a given [=/origin=] |issuer|, run the following steps:
+
+1. Let |commitment| be the result of [=look up the key commitments|looking up the key commitments=] for |issuer|.
+1. If |commitment| is null, return null.
+1. Let |chosenKey| be null.
+1. Let |currentTime| be the current date and time.
+1. For each |key| in |commitment|["keys"], run the following steps:
+    1. If |key|["expiry"] is less than |currentTime|, skip processing this element.
```

Comment:
What does `skip processing this element` mean? To exit the loop (then write `continue`) or to exit the function (then write `return`)?